src/wat-writer.cc

```diff
@@ -485,6 +485,15 @@ void WatWriter::WriteConst(const Const& const_) {
       break;
     }
 
+    case Type::V128: {
+      int32_t i32Array[4];
+      memcpy(&i32Array, &(const_.v128_bits), sizeof(v128));
```

Comment:
Remove and just use `const_.v128_bits.v[0]`, etc. below.

Reply:
Agree.